### ISSUE TITLE
Why THREE.Line can't be merged with THREE.Mesh ?

### ISSUE THREAD

**Issue by eliasasaid1** [mrdoob/three.js#14555]
Hello,

I saw in an old thread that we can't merge THREE.Line with THREE.Mesh but can I ask why? when I add a lot of lines into my application it will cause performance issue and that's why I need to merge them with the mesh. Any possibility of fixing that in the next updates?

**Comment by oguzeroglu** [comment]
I don't think this is possible because the way they are rendered are different. THREE.Mesh uses **gl.TRIANGLES** as the draw mode whereas THREE.Line uses **gl.LINES**.
